app/models/ems_refresh/parsers/openshift.rb

```diff
@@ -30,12 +30,12 @@ def parse_project(project)
       def parse_route(route)
         new_result = parse_base_item(route)
         new_result.merge!(
-            # TODO: persist tls
-            :host_name    => route.host,
-            :labels       => parse_labels(route),
-            # TODO: this part needs to be modified to service_id instead
-            :service_name => route.serviceName,
-            :path         => route.path
+          # TODO: persist tls
+          :host_name    => route.spec.host,
+          :labels       => parse_labels(route),
+          # TODO: this part needs to be modified to service_id instead
+          :service_name => route.spec.to,
```

Comment:
Have you tested this? I expected this to be something like:

    route.spec.to.name

Or even better:

    route.spec.try(:to).try(:name)

Also I'd check that `route.spec.try(:to).try(:kind)` is `Service` (otherwise it won't make sense to store this value here).